extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -98,8 +98,11 @@ export class AmpAd3PImpl extends AMP.BaseElement {
 
     setupA2AListener(this.win);
 
-    /** @private {?Element|undefined} */
+    /** @private {?string|undefined} */
     this.container_ = undefined;
```

Comment:
should we just make it `null`, to reduce the possible value for the case of "no container".